Makefile

```diff
@@ -0,0 +1,12 @@
+TAG := $(notdir $(shell pwd))
+
+.PHONY: all
+all: build
+
+.PHONY: build
+build: Dockerfile
+	docker build -t '${TAG}' .
+
+.PHONY: run
+run: build
+	docker run --rm -it --name='${TAG}' '${TAG}'
```

Comment:
where this line? 

https://github.com/fulcanelly/auclean-bot/blob/e4a3c6ae4c2346435f087870896d61fd81602104/docker-compose.yml#L5

Reply:
```sh
# run in background
make daemon

# stop
make stop-daemon
```